src/main/resources/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/generic-whitelist

```diff
@@ -32,6 +32,12 @@ method groovy.lang.Closure ncurry int java.lang.Object[]
 method groovy.lang.Closure setDelegate java.lang.Object
 method groovy.lang.Closure setResolveStrategy int
 method groovy.lang.GString plus java.lang.String
+method groovy.lang.NumberRange by int
+method groovy.lang.NumberRange getStepSize
```

Comment:
@haridsv `NumberRange` does not exist in Groovy 2.4.x as far as I can tell, which is what Jenkins is currently using, so these entries don't make sense. See https://github.com/jenkinsci/script-security-plugin/pull/268.